taskcluster/mozillavpn_taskgraph/transforms/scriptworker.py

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Perhaps it's not ready for use yet - but this strikes me as something that belongs in https://github.com/mozilla-releng/mozilla-taskgraph/ at some point.

Reply:
100% agreed! I'm going to do one phase at a time though, so you can expect another PR in the future.